.idea/codeStyles/Project.xml

```diff
@@ -8,6 +8,7 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
       <option name="IMPORT_LAYOUT_TABLE">
         <value>
+          <package name="" withSubpackages="true" static="false" module="true" />
```

Comment:
IDK what this does, but was it intended?

Reply:
It’s a new setting in IntelliJ 2924.x to sort the order of imports from a module. Required because I touched the code formatting settings (to use the project config)